frontend/src/global_styles/content/modules/_avatars.sass

```diff
@@ -28,4 +28,12 @@
 
 .user-avatar--multi-line
   display: block
-  margin-bottom: 2px
\ No newline at end of file
+  margin-bottom: 2px
+
+.avatar
```

Comment:
if this is a BEM block you want to keep I'd suggest renaming it to `op-avatar` and putting it into its own file

Reply:
Might be hard to do right now with all the legacy stuff though